src/renderer/components/instance/UnsavedDataWarning.ts

```diff
@@ -0,0 +1,46 @@
+import { css, customElement, html, LitElement, TemplateResult } from "lit-element";
+import "@vaadin/vaadin-button";
+
+@customElement("unsaved-data-warning")
+export class UnsavedDataWarning extends LitElement {
+	public static styles = css`
+		:host {
+			background-color: rgb(54, 54, 54);
+			border-radius: 0.25em;
+			padding: 0 10px;
+		}
+
+		span {
+			display: inline-block;
+			color: white;
+		}
+
+		#save-button {
+			margin-left: 10px;
+		}
+	`;
+
+	protected render(): TemplateResult {
+		return html`
```

Comment:
Codacy found an issue: [Unsafe return of an any typed value](https://app.codacy.com/manual/lukechu10/Minecraft-Box-Launcher/pullRequest?prid=6041480)